packages/openjdk.vm/openjdk.vm.nuspec

```
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>openjdk.vm</id>
-    <version>21.0.1</version>
+    <version>21.0.1.20240202</version>
```

To be consistent with [python3.vm](https://github.com/mandiant/VM-Packages/blob/main/packages/python3.vm/python3.vm.nuspec#L5) and [nodejs.vm](https://github.com/mandiant/VM-Packages/blob/main/packages/nodejs.vm/nodejs.vm.nuspec#L5) this should be 
```suggestion
    <version>0.0.0.20240202</version>
```

This means we need to remove the current package/s from myget before merging this PR. @emtuls can you update the version and I'll take care of removing the packages and merging this PR?

But this may need further discussion